package/MDAnalysis/analysis/dielectric.py

```diff
@@ -0,0 +1,140 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Dielectric --- :mod:`MDAnalysis.analysis.dielectric`
+===========================================================
+
+A tool to compute the trajectory average static dielectric
+constant for a system simulated in tin foil boundary conditions.
+
+References
+----------
+  Mol. Phys. 1983, vol. 50, No. 4, p. 848
+  DOI: 10.1080/00268978300102721
+"""
+from __future__ import division
+
+import numpy as np
+from numpy.testing import assert_almost_equal
+
+from MDAnalysis.analysis.base import AnalysisBase
+
+class DielectricConstant(AnalysisBase):
+    """Dielectric constant computation
+    DielectricConstant(selection, temperature=298.15)
+
+    Parameters
+    ----------
+    selection : `AtomGroup` object
+      Any atomgroup
+
+    Keywords
+    --------
+    temperature : float
+          Temperature (Kelvin) at which the system has been simulated [298.15]
+
+    start : int
+          The frame to start at [0]
+    stop : int
+          The frame to end at [-1]
+    step : int
+          The step size through the trajectory in frames [0]
```

Comment:
Ok will change that. Same wrong info is also in the rdf module docstrings (which is where I copied/pasted from).